src/controllers/clientController.js

```diff
@@ -0,0 +1,124 @@
+const Client = require('../models/client');
+const Dish = require('../models/dish');
+
+exports.createClient = async (req, res) => {
+  try {
+    const { user, address, contactNumber } = req.body;
+
+    const client = new Client({
+      user,
+      address,
+      contactNumber,
+    });
+
+    const createdClient = await client.save();
+
+    res.status(201).json(createdClient);
+  } catch (error) {
+    res.status(500).json({ error: 'Server error' });
+  }
+};
+
+exports.getClient = async (req, res) => {
+  try {
+    const clientId = req.body.id;
+    const client = await Client.findById(clientId);
+
+    if (!client) {
+      res.status(404).json({ error: 'Client not found' });
+    }
+
+    res.status(200).json(client);
+  } catch (error) {
+    res.status(500).json({ error: 'Server error' });
+  }
+};
```

Comment:
This is just a note for the feature, we can keep the code as it is for now.

What we are doing now is getting the id from the body, but later we will have jwt inside the cookie we are going to use that to get the client details.

Reply:
yes i did that for testing

Reply:
great job özlem....